src/main/java/org/opensearch/security/securityconf/DynamicConfigModelV7.java

```diff
@@ -358,6 +375,17 @@ private void buildAAA() {
             }
         }
 
+        Settings oboSettings = getDynamicOnBehalfOfSettings();
+        if (oboSettings.get("signing_key") != null && oboSettings.get("encryption_key") != null) {
```

Comment:
This logic is duplicated with the parts of the OnBehalfOfAuthenticator / JWT vendor, this logic should be in one place in case it changes

Reply:
https://github.com/opensearch-project/security/issues/3238 is the issue to capture this follow up